frontend/packages/kubevirt-plugin/src/components/vmtemplates/vmtemplates.tsx

```diff
@@ -0,0 +1,136 @@
+
+const tableColumnClass = classNames('col-lg-2', 'col-sm-4', 'col-xs-4');
+const tableColumnClassHiddenOnSmall = classNames('col-lg-2', 'hidden-sm', 'hidden-xs');
+
+const tableColumnClasses = [
+  tableColumnClass,
+  tableColumnClass,
+  tableColumnClass,
+  tableColumnClassHiddenOnSmall,
+  tableColumnClassHiddenOnSmall,
+  tableColumnClassHiddenOnSmall,
+  Kebab.columnClass,
+];
+
+const VmTempleateTableHeader = () => {
+  return [
+    {
+      title: 'Name',
+      sortField: 'metadata.name',
+      transforms: [sortable],
+      props: { className: tableColumnClasses[0] },
+    },
+    {
+      title: 'Namespace',
+      sortField: 'metadata.namespace',
+      transforms: [sortable],
+      props: { className: tableColumnClasses[1] },
+    },
+    {
+      title: 'Description',
+      sortField: 'metadata.annotations.description',
+      transforms: [sortable],
+      props: { className: tableColumnClasses[2] },
+    },
+    {
+      title: 'Source',
+      props: { className: tableColumnClasses[3] },
+    },
+    {
+      title: 'OS',
+      props: { className: tableColumnClasses[4] },
+    },
+    {
+      title: 'Flavor',
+      props: { className: tableColumnClasses[5] },
+    },
+    {
+      title: '',
+      props: { className: tableColumnClasses[6] },
+    },
+  ];
+};
+VmTempleateTableHeader.displayName = 'VmTemplateTableHeader';
+
+const VmTemplateTableRow = ({ obj: template, index, key, style }) => {
+  const os = getTemplateOperatingSystems([template])[0];
+
+  return (
+    <TableRow id={template.metadata.uid} index={index} trKey={key} style={style}>
+      <TableData className={tableColumnClasses[0]}>
+        <ResourceLink
+          kind={kind}
+          name={getName(template)}
+          namespace={getNamespace(template)}
+          title={template.metadata.uid}
+        />
+      </TableData>
+      <TableData className={tableColumnClasses[1]}>
+        <ResourceLink
+          kind="Namespace"
+          name={getNamespace(template)}
+          title={getNamespace(template)}
+        />
+      </TableData>
+      <TableData className={tableColumnClasses[2]}>
+        {_.get(template.metadata, 'annotations.description', DASH)}
+      </TableData>
+      <TableData className={tableColumnClasses[3]}>
+        <TemplateSource template={template} />
+      </TableData>
+      <TableData className={tableColumnClasses[4]}>{os ? os.name || os.id : DASH}</TableData>
+      <TableData className={tableColumnClasses[5]}>{getTemplateFlavors([template])[0]}</TableData>
+      <TableData className={tableColumnClasses[6]}>
+        <ResourceKebab actions={menuActions} kind={kind} resource={template} />
+      </TableData>
+    </TableRow>
+  );
+};
+VmTemplateTableRow.displayName = 'VmTemplateTableRow';
+
+const VirtualMachineTemplates = (props) => (
+  <Table
+    {...props}
+    aria-label={labelPlural}
+    Header={VmTempleateTableHeader}
+    Row={VmTemplateTableRow}
+    virtualize
```

Comment:
Please note, the `virtualize` prop should default to `true` (based on discussion with @priley86).

Reply:
:+1: